docs/README-dev.md

```diff
@@ -217,22 +221,30 @@ Next, bring in the Narrative and Bootstrap repos (as yourself):
     git clone http://github.com/kbase/bootstrap
     git clone http://github.com/kbase/ui-common
 
+# note that as of 2/10/15, narrative-test.kbase.us is using the
+# develop branch of both narrative and ui-common
```

Comment:
The 'staging' branch of both narrative and ui-common is deployed on narrative-test.kbase.us